### PR TITLE
[Docs] LDAP authentication for YCQL

### PR DESCRIPTION
**Pull request by pkj415**
@netlify /latest/secure/authentication/ldap-authentication-ycql

### CODE REVIEW

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+   </td>
+   <td><strong>Default value</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>ycql_ldap_base_dn
+   </td>
+   <td>Specifies the base directory to begin the user name search
+   </td>
+   <td>''
```
Comment (m-iancu) [inline docs/content/latest/secure/authentication/ldap-authentication-ycql.md:151]: nit: maybe these empty strings (this an below) would look clearer as code snippets `''` vs ''.

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+    ```sh
+    --use_cassandra_authentication=true --ycql_use_ldap=true --ycql_ldap_server=ldap://ldap.forumsys.com:389 --ycql_ldap_user_prefix=uid= --ycql_ldap_user_suffix=, dc=example, dc=com --ycql_ldap_users_to_skip_csv=cassandra
+    ```
+
+    {{< note title="Note" >}}
+In the above sample configuration, we are using an [online LDAP test server](https://www.forumsys.com/tutorials/integration-how-to/ldap/online-ldap-test-server/) for setting up the LDAP authentication with YugabyteDB.
+    {{< /note >}}
+
+    The --ycql_ldap_users_to_skip_csv=cassandra gflag allows access to the user `cassandra` with password authentication. This allows the administrator to log in for setting up the roles (and permissions) for the LDAP users.
```
Comment (m-iancu) [inline docs/content/latest/secure/authentication/ldap-authentication-ycql.md:202]: nit: I think --ycql_ldap_users_to_skip_csv=cassandra would be clearer if in a code snippet (`--ycql_ldap_users_to_skip_csv=cassandra`)

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+
+1. Start the YugabyteDB cluster.
+
+1. Open the YSQL shell (ysqlsh), specifying the `cassandra` user and prompting for the password.
```
Comment (m-iancu) [inline docs/content/latest/secure/authentication/ldap-authentication-ycql.md:206]: nit: YSQL shell (ysqlsh) --> YCQL Shell (`ycqlsh`)

the code block part is optional, up to you.

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+    cassandra@ycqlsh> create role adam with login=true;
+    ```
+
+1. Connect using LDAP authentication.
+
+    Connect ysqlsh using the `riemann` LDAP user and password specified in the [Online LDAP Test Server](https://www.forumsys.com/tutorials/integration-how-to/ldap/online-ldap-test-server/) page. 
+
+    ```sh
+    $ ./ycqlsh -u riemann
+    ```
```
Comment (m-iancu) [inline docs/content/latest/secure/authentication/ldap-authentication-ycql.md:235]: nit: might be nice to have the output/shell after this command works. 
E.g. like the example above but with the new username (or w/e ycqlsh actually outputs):
```
Connected to local cluster at 127.0.0.1:9042.
[ycqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
Use HELP for help.
riemann@ycqlsh>
```

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+    $ ./ycqlsh -u cassandra
+    ```
+
+    When prompted for the password, enter `cassandra` (default password of `cassandra` user). You should be able to login and see a response like below.
+
+    ```output
+    Connected to local cluster at 127.0.0.1:9042.
+    [ycqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
+    Use HELP for help.
+    cassnadra@ycqlsh>
```
Comment (m-iancu) [inline docs/content/latest/secure/authentication/ldap-authentication-ycql.md:218]: typo: cassnadra -> cassandra

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md (hunk 1 above)
Comment (pkj415) [reply docs/content/latest/secure/authentication/ldap-authentication-ycql.md:151]: done

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md (hunk 2 above)
Comment (pkj415) [reply docs/content/latest/secure/authentication/ldap-authentication-ycql.md:202]: done

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md (hunk 3 above)
Comment (pkj415) [reply docs/content/latest/secure/authentication/ldap-authentication-ycql.md:206]: done

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md (hunk 5 above)
Comment (pkj415) [reply docs/content/latest/secure/authentication/ldap-authentication-ycql.md:218]: nice catch!

File: docs/content/latest/secure/authentication/ldap-authentication-ycql.md (hunk 4 above)
Comment (pkj415) [reply docs/content/latest/secure/authentication/ldap-authentication-ycql.md:235]: done

**Review by m-iancu — APPROVED** [review]
LGTM, thanks @pkj415!

**Pull request merged**